yarn.lock

```diff
@@ -1478,7 +1478,7 @@
   version "0.1.11"
   resolved "https://codeload.github.com/snapshot-labs/snapshot.js/tar.gz/master"
   dependencies:
-    "@ensdomains/content-hash" "^2.5.3"
```

Comment:
Any reason for changing this yarn.lock?

Reply:
Actually, it seems my node version was not correct.